test/assignment.coffee

```diff
@@ -916,36 +600,6 @@ test "#4566: destructuring with nested default values", ->
   {e: {f = 5} = {}} = {}
   eq 5, f
 
-test "#4674: _extends utility for object spreads 1", ->
-  eqJS(
-    "{a, b..., c..., d}"
-    """
-      var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
-
-      _extends({a}, b, c, {d});
-    """
-  )
-
-test "#4674: _extends utility for object spreads 2", ->
-  _extends = -> 3
-  a = b: 1
-  c = d: 2
-  e = {a..., c...}
-  eq e.b, 1
-  eq e.d, 2
```

Comment:
These `_extends` tests are the only ones you’ve removed, correct? Are there any that you changed while moving them into the new file?